sapi/fpm/fpm/fpm.c

```diff
@@ -53,18 +53,18 @@ enum fpm_init_return_status fpm_init(int argc, char **argv, char *config, char *
 	fpm_globals.run_as_root = run_as_root;
 	fpm_globals.force_stderr = force_stderr;
 
-	if (0 > fpm_php_init_main()           ||
-	    0 > fpm_stdio_init_main()         ||
-	    0 > fpm_conf_init_main(test_conf, force_daemon) ||
-	    0 > fpm_unix_init_main()          ||
-	    0 > fpm_scoreboard_init_main()    ||
-	    0 > fpm_pctl_init_main()          ||
-	    0 > fpm_env_init_main()           ||
-	    0 > fpm_signals_init_main()       ||
-	    0 > fpm_children_init_main()      ||
-	    0 > fpm_sockets_init_main()       ||
-	    0 > fpm_worker_pool_init_main()   ||
-	    0 > fpm_event_init_main()) {
+	if (fpm_php_init_main() != SUCCESS ||
```

Comment:
note bukka usually prefers when operands are reversed :-) but I ll leave up to him to decide on this.